engine/classes/ElggInstaller.php

```diff
@@ -64,6 +64,11 @@ public function __construct() {
 		if (!isset($CONFIG)) {
 			$CONFIG = new stdClass;
 		}
+		
+		global $_ELGG;
```

Comment:
I think I rebased at one point and added it. If it shouldn't be here I'll just remove it...